test/e2e/specs/scaleupdown.go

```diff
@@ -0,0 +1,216 @@
+package specs
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/ghodss/yaml"
+	apiappsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/util/wait"
+
+	internal "github.com/openshift/openshift-azure/pkg/api"
+	"github.com/openshift/openshift-azure/pkg/api/2018-09-30-preview/api"
+	"github.com/openshift/openshift-azure/pkg/util/randomstring"
+	"github.com/openshift/openshift-azure/pkg/util/ready"
+	"github.com/openshift/openshift-azure/test/clients/azure"
+	"github.com/openshift/openshift-azure/test/clients/openshift"
+)
+
+var _ = Describe("Scale Up/Down E2E tests [ScaleUpDown] [Real] [LongRunning] [EveryPR]", func() {
```

Comment:
nit: remove the space between the tags

Reply:
We should be able to run this test in both the fake and the real RP. Real RP testing is blocked until they rebase to 3.11 so I would suggest make this work for the fake RP and open a follow-up issue to enable it for the real RP.